apps/web/components/CheckoutAddress/CheckoutAddress.vue

```diff
@@ -2,21 +2,29 @@
   <div data-testid="checkout-address" class="md:px-4 py-6">
     <div class="flex justify-between items-center">
       <h2 class="text-neutral-900 text-lg font-bold mb-4">{{ heading }}</h2>
-      <SfButton v-if="savedAddress" size="sm" variant="tertiary" @click="open">
+      <SfButton v-if="addresses.length > 0" size="sm" variant="tertiary" @click="edit">
```

Comment:
**suggestion (non-blocking)**: I think it's nice to have a `savedAddress` to clearly express what it means if `length > 0`.

We also check this in lines 10 and 26 in combination with authentication. When dealing with a combined condition, I feel like having a named function improves readability even more.